### ISSUE TITLE
`SpaceBeforeOpeningBrace` fails

### ISSUE THREAD

**Issue by modestukasai** [CodeNarc/CodeNarc#332]
Rule fails for this case:
```
result.containsAll([{ it.name == 'Sweden' } as CountryRequest, { it.name == 'Denmark' } as CountryRequest])
```
```
The opening brace for the closure in class xxxx is not preceded by a space or whitespace
```

Adding `checkClosureMapEntryValue` as false, does not help

**Comment by chrismair** [comment]
Okay, that makes sense. That is a List, rather than a Map.

This exposes some inconsistencies -- not just in handling List vs Map, but also between the *SpaceBeforeOpeningBrace* and *SpaceAfterOpeningBrace* rules.

**Comment by chrismair** [comment]
Fixed for CodeNarc v1.2
